src/parsers/kovork.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const moment = require('moment');
+
+function parseData(html, date) {
```

Comment:
It still only look for FB post with date in right format, so I prefer sam general file name like `fb` or `fb-post`,
but I could live with that.